data/core/units/elves/Sylph.cfg

```diff
@@ -8,15 +8,15 @@
     small_profile="portraits/elves/sylph.webp~CROP(26,30,420,420)"
     profile="portraits/elves/sylph.webp~RIGHT()"
     halo=halo/elven/shyde-stationary-halo[1~6].png:150
-    hitpoints=68
+    hitpoints=60
     movement_type=woodlandfloat
     movement=6
     experience=200
     level=4
     alignment=neutral
     advances_to=null
     {AMLA_DEFAULT}
-    cost=135
+    cost=120
     usage=mixed fighter
     description= _ "Tremendously powerful in unfathomable ways, the sage-like Sylphs are masters of manipulating the bridge between the mundane and arcane worlds. Long years spent peering into the ethereal realm have eroded the ability of these elves to view the physical world; in return, they are granted an abstract sight, gaining the ability to view one or even several different aspects of reality’s essence. Like the many shards of a broken mirror, the myriad fractures of the material world reflect the light of the arcane through its many different facets. Careful practice allows one to follow these strands of light from pane to pane, observing how the outcome of physical reality evolves with each choice made of free will. While direct weaving of the connecting fabric is usually impossible to achieve, indirect manipulation is feasible by machination in the corporeal plane where the reflection of the arcane plane is strongest. The ability of a Sylph to locate these ‘reflection pools’ and steer them is one of her greatest — and most feared — abilities."
```

Comment:
"Tremendously"...No, change this word to lesser type of Powerful. You are nerfing her a good deal it seems so this word cannot fit.

Reply:
I think the Sylphs can still be tremendously powerful in many other ways than in warfare.

Reply:
> "Tremendously"...No, change this word to lesser type of Powerful. You are nerfing her a good deal it seems so this word cannot fit.
> hmm, 45 (9-5) for a level 4...Would have prefered 10-5 but this works too.

I did still mean for the Sylph to be quite powerful. I'd hope that her high mobility, exceptional defense, and powerful slow can compensate for her lower raw damage output than the Great Mage and Ancient Lich. But perhaps we can buff up her gossamer a little to ensure she still feels "tremendously" powerful.